src/DefaultProgressBar.jsx

```diff
@@ -0,0 +1,80 @@
+import PropTypes from 'prop-types';
+import { forbidExtraProps } from 'airbnb-prop-types';
+import React from 'react';
+
+import { withStyles, withStylesPropTypes } from 'react-with-styles';
+
+const propTypes = forbidExtraProps({
+  ...withStylesPropTypes,
+  orientation: PropTypes.string,
+  disabled: PropTypes.bool,
+  style: PropTypes.any,
```

Comment:
This is a different prop from `styles`, and is passed in through the slider component. I agree it's not great, but this was what was in the internal version. Since this is just supposed to be a port (rather than a rewrite - I want to make sure I stay in scope!) I will just leave it for now. In the future, this is definitely something that should be fixed. 

In the meantime, I will change to `PropTypes.object`.

Reply:
eesh, ok - we don't want to publish a component tho that has a `style` prop - i'd prefer to not support that use case initially if we don't want to rewrite it.